deploy-jobs-template.yml

```diff
@@ -303,13 +318,43 @@ jobs:
                       },
                       {
                         "name": "FUNCTIONS_WORKER_RUNTIME",
-                        "value": "dotnet",
+                        "value": "$(FunctionsWorkerRuntime)",
```

Comment:
Need to time application of FunctionsWorkerRuntime through deployment fully intend an Enabling Release which will require a deployment to test at least, if not a code change or two